README.nl.md

```diff
@@ -0,0 +1,32 @@
+- [English](README.md)
+- [Dutch](README.nl.md)
+- [日本語](README.ja.md)
+- [한국어](README.ko.md)
+- [Bahasa Indonesia](README.in.md)
+- [Español](README.es.md)
+- [简体中文](README.zh.md)
+
+# Laten-we-gearresteerd-worden project
+
+In Japan werd een schoolmeisje opgepakt door de politie voor het plaatsen van een link naar een website met een eindeloze lustreeks in JavaScript, zoals dit:
```

Comment:
`eindeloze lustreeks` -> `oneindige lus` or otherwise just keep it as `infinite loop`?